src/main/flight/pid.c

```diff
@@ -778,6 +811,13 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
 
     // Dynamic i component,
     // gradually scale back integration when above windup point
```

Comment:
This comment is for dynCi variable so move it down before `const float dynCi = ...`

Reply:
Yes, missed that - thanks!